force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +314,11 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
+        Book newBook = new Book('A Brief History of Time', 'Stephen Hawking');
         // Call the getBookDetails method on the book object to get the details of the book
-
+        String detailsBook = newBook.getBookDetails();
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return detailsBook; 
```

Comment:
good work. 

The challenge is to do this with one line.  You can, and I don't think you lose readability. Your method returns a String and getBookDetails returns a String.